tests/config/basic/webdavDatastore.yaml

```diff
@@ -1,6 +1,6 @@
 datastore:
   cls: lsst.daf.butler.datastores.webdavDatastore.WebdavDatastore
-  root: https://some.webdav.server/butlerRoot
+  root: http://localhost:8080/butlerRoot
```

Comment:
Maybe add a comment to the YAML here noting that this is an explicit URL for the test server rather than something random.